backend/CircleForms/Startup.cs

```diff
@@ -40,7 +42,8 @@ public void ConfigureServices(IServiceCollection services)
         services.Configure<OsuApiConfig>(config);
         services.Configure<SuperAdminsId>(Configuration.GetSection("SuperAdmins"));
 
-        services.AddAutoMapper(x => x.AddProfile(new SubmissionContractToSubmissionMapper()));
+        services.AddAutoMapper(x =>
+            x.AddProfiles(new Profile[] {new SubmissionContractToSubmissionMapper(), new PostToPostRedisMapper()}));
```

Comment:
Might as well just move all mapping profiles into one class, for example `MappingProfiles`. Having them separated seems redundant to me